deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py

```diff
@@ -0,0 +1,217 @@
+import argparse
```

Comment:
Please review the PEP 8 compliance and docstring formats for all Python files.
```
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:1:1: D100 Missing docstring in public module
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:1:1: I001 [*] Import block is un-sorted or un-formatted
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:14:5: D417 Missing argument description in the docstring for `format_directory`: `directory`
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:15:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:16:5: D410 [*] Missing blank line after section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:16:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:18:5: D411 [*] Missing blank line before section ("Returns")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:24:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:25:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:30:53: W291 [*] Trailing whitespace
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:31:1: W293 [*] Blank line contains whitespace
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:33:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:34:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:41:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:42:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:52:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:53:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:75:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:76:5: D410 [*] Missing blank line after section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:76:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:79:5: D411 [*] Missing blank line before section ("Returns")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:85:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:86:5: D410 [*] Missing blank line after section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:86:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:88:5: D411 [*] Missing blank line before section ("Returns")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:101:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:102:5: D410 [*] Missing blank line after section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:102:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:104:5: D411 [*] Missing blank line before section ("Returns")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:107:47: F541 [*] f-string without any placeholders
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:110:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:111:5: D410 [*] Missing blank line after section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:111:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:113:5: D411 [*] Missing blank line before section ("Returns")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:119:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:120:5: D410 [*] Missing blank line after section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:120:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:123:5: D411 [*] Missing blank line before section ("Returns")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:129:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:130:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:144:5: D205 1 blank line required between summary line and description
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:145:5: D411 [*] Missing blank line before section ("Args")
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:157:17: F541 [*] f-string without any placeholders
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:167:5: D103 Missing docstring in public function
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:168:65: W291 [*] Trailing whitespace
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:173:30: F541 [*] f-string without any placeholders
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:174:1: W293 [*] Blank line contains whitespace
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:176:30: F541 [*] f-string without any placeholders
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:177:1: W293 [*] Blank line contains whitespace
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:179:30: F541 [*] f-string without any placeholders
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:182:30: F541 [*] f-string without any placeholders
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:185:30: F541 [*] f-string without any placeholders
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:188:30: F541 [*] f-string without any placeholders
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:191:30: F541 [*] f-string without any placeholders
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:192:1: W293 [*] Blank line contains whitespace
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:195:1: W293 [*] Blank line contains whitespace
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:197:30: F541 [*] f-string without any placeholders
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:201:30: F541 [*] f-string without any placeholders
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:202:1: W293 [*] Blank line contains whitespace
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:208:5: D103 Missing docstring in public function
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:213:9: B007 Loop control variable `n` not used within loop body
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:213:45: W291 [*] Trailing whitespace
deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py:217:11: W292 [*] No newline at end of file
 
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/4fee113a6384ad0a9d5dd204ecf37708f0632be5